TW-286 proxy session docs (#3793)
postmanlabs/postman-docs: src/pages/docs/sending-requests/capturing-request-data/capturing-https-traffic.md

    name: "Sending requests"
    url: "/docs/sending-requests/requests/"
  - type: section
    name: "Additional Resources"
  - type: subtitle
    name: "Videos"
  - type: link
    name: "Capture API Calls With a Proxy"
    url: "https://youtu.be/bjrCHUITZ3k"
  - type: subtitle
    name: "Related Blog Posts"
  - type: link
    name: "Reverse engineering an API"
    url: "https://medium.com/better-practices/reverse-engineering-an-api-403fae885303"
  - type: link
    name: "Using Postman Proxy to Capture and Inspect API Calls from iOS or Android Devices"
    url: "https://blog.postman.com/using-postman-proxy-to-capture-and-inspect-api-calls-from-ios-or-android-devices/"
  - type: link
    name: "Capture Responses Using the Postman Proxy"
    url: "https://blog.postman.com/capture-responses-using-the-postman-proxy/"

warning: false

---

In addition to capturing HTTP traffic, you can use the Postman's built-in proxy to inspect HTTPS communication from your Android, iOS, Linux, macOS, and Windows devices.

## Contents

* [Capturing HTTPS traffic](#capturing-https-traffic)
* [Troubleshooting certificate issues](#troubleshooting-certificate-issues)

## Capturing HTTPS traffic

> You must install the `postman-proxy-ca.crt` certificate on your device to be able to capture secure HTTP traffic. First, [set up the proxy in Postman](/docs/sending-requests/capturing-request-data/capturing-http- requests/). Then follow the instructions below to install the required security certificate on the target devices.

Reviewer: ```suggestion
You must install the `postman-proxy-ca.crt` certificate on your device to be able to capture secure HTTP traffic. First, [set up the proxy in Postman](/docs/sending-requests/capturing-request-data/capturing-http-requests/). Then follow the instructions below to install the required security certificate on the target devices.
```

Author: (already fixed)

    ![download OpenSSL installer](https://assets.postman.com/postman-docs/windows-oppssl-environment-variables-bin.jpg)

1. Select **OK** to add the folder directory. Then select **OK** to confirm changes and close the remaining windows.
1. Open a command prompt. To do this, open the Windows **Start** menu, search for **cmd**, and select **Open**. Enter the command `openssl version` to confirm that installation was successful. You should see output similar to the following:

    ``OpenSSL 1.1.1l  24 Aug 2021``

#### Installing the security certificate on Windows

> Before you begin, make sure to [install the OpenSSL module](#installing-openssl-on-windows) so Postman can generate the certificate.

1. In Windows File Explorer, navigate to the `%APPDATA%\Postman\proxy` folder. Typically, the folder will be located at `C:\Users\<user>\AppData\Roaming\Postman\proxy`.

1. Right-click on the **postman-proxy-ca.crt** file and select **Install Certificate**.

    [![Select crt file](https://assets.postman.com/postman-docs/windows-install-crt-aa.jpeg)](https://assets.postman.com/postman-docs/windows-install-crt-aa.jpeg)

> 1. Select **Local Machine** and select **Next**. This action requires Administrator permissions—select **Yes** to proceed.

Reviewer: ```suggestion
1. Select **Local Machine** and select **Next**. This action requires Administrator permissions. Select **Yes** to proceed.
```

    * **Windows:** `C:\Users\<user>\AppData\Roaming\Postman\proxy`
    * **Linux:** `~/.config/Postman/proxy`

1. Go to **Settings > Profile Downloaded**, and then select **Install**. Enter your passcode to proceed.
1. A security warning displays. Select **Install**.
1. After the certificate is installed, select **Done**.
1. Go to **Settings > General > About > Certificate Trust Settings**.
1. Enable full trust for Postman’s root certificate, and select **Continue** to complete the installation.

### Android

> The certificate installation process may differ depending on your device and Android version.

1. Download the `postman-proxy-ca.crt` certificate to the Android device. You can find the certificate file on your computer in the following location:

    * **macOS:** `~/Library/Application Support/Postman/proxy`
    * **Windows:** `C:\Users\<user>\AppData\Roaming\Postman\proxy`
    * **Linux:** `~/.config/Postman/proxy`

1. Open the **Settings** app and go to **Security > Encryption & credentials**.
1. Select **Install a certificate** and select the **CA Certificate** option.
1. A security warning displays. Select **Install anyway** to proceed.

> 1. Browse for and select the `postman-proxy-ca.crt` certificate file. A message displays stating that the certificate is installed. You can now capture traffic through a web browser on Android.

Reviewer: ```suggestion
1. Browse for and select the `postman-proxy-ca.crt` certificate file. You will see a message that the certificate is installed. You can now capture traffic through a web browser on Android.
```

> **Need to capture requests from an Android app?** To capture requests from an Android app, you need to add a network security configuration file to your app to trust the `postman-proxy-ca.crt` certificate. For more information, see [Trust additional CAs](https://developer.android.com/training/articles/security-config#TrustingAdditionalCas) on the Android Developers portal.

Reviewer: Maybe format this as a note to make it pop a little more?

```suggestion
> **Need to capture requests from an Android app?** To capture requests from an Android app, you need to add a network security configuration file to your app to trust the `postman-proxy-ca.crt` certificate. For more information, see [Trust additional CAs](https://developer.android.com/training/articles/security-config#TrustingAdditionalCas) on the Android Developers portal.
```
